src/screens/OtherProfile/index.js

```diff
@@ -53,82 +53,10 @@ import {generateAnonProfileOtherProfile} from '../../service/anonymousProfile';
 import {getSingularOrPluralText} from '../../utils/string/StringUtils';
 import {linkContextScreenParamBuilder} from '../../utils/navigation/paramBuilder';
-import {sendAnonymousDMOtherProfile, sendSignedDMOtherProfile} from '../../service/chat';
 import {setFeedByIndex, setOtherProfileFeed} from '../../context/actions/otherProfileFeed';
-import {trimString} from '../../utils/string/TrimString';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
-import StorageUtils from '../../utils/storage';
-import useCoreFeed from '../FeedScreen/hooks/useCoreFeed';
-import useCreateChat from '../../hooks/screen/useCreateChat';
-import {ANON_PM, SIGNED} from '../../hooks/core/constant';
 
-const {width, height} = Dimensions.get('screen');
-// let headerHeight = 0;
-
-const BioAndChat = (props) => {
-  const {
-    isAnonimity,
-    bio,
-    openBio,
-    isSignedMessageEnabled,
-    showSignedMessageDisableToast,
-    loadingGenerateAnon,
-    avatarUrl,
-    anonProfile,
-    onSendDM,
-    setDMChat,
-    loadingSendDM,
-    dmChat,
-    username,
-    toggleSwitch,
-    isAnonimityEnabled
-  } = props;
-  return (
-    <View style={styles.bioAndSendChatContainer(isAnonimity)}>
-      <View style={styles.containerBio}>
-        {bio === null || bio === undefined ? (
-          <Text style={styles.bioText(isAnonimity)}>Send a message</Text>
-        ) : (
-          <Pressable onPress={openBio}>
-            <Text linkStyle={styles.seeMore} style={styles.bioText(isAnonimity)}>
-              {trimString(bio, 121)}{' '}
-              {bio.length > 121 ? <Text style={{color: colors.blue}}>see more</Text> : null}
-            </Text>
-          </Pressable>
-        )}
-      </View>
-      <TouchableOpacity
-        disabled={isSignedMessageEnabled}
-        activeOpacity={1}
-        onPress={showSignedMessageDisableToast}>
-        <TextAreaChat
-          isAnonimity={isAnonimity}
-          loadingAnonUser={loadingGenerateAnon}
-          avatarUrl={avatarUrl}
-          anonUser={anonProfile}
-          placeholder="Send a direct message"
-          disabledInput={!isSignedMessageEnabled}
-          onSend={onSendDM}
-          onChangeMessage={setDMChat}
-          disabledButton={loadingSendDM || !isSignedMessageEnabled || loadingGenerateAnon}
-          defaultValue={
-            isSignedMessageEnabled ? dmChat : `Only users ${username} follows can send messages`
-          }
-        />
-      </TouchableOpacity>
-      <TouchableOpacity onPress={toggleSwitch} style={styles.toggleSwitchContainer}>
-        <ToggleSwitch
-          value={isAnonimity}
-          onValueChange={toggleSwitch}
-          labelLeft={
-            isAnonimityEnabled || !isSignedMessageEnabled ? 'Anonymity' : 'Anonymity disabled'
-          }
-          styleLabelLeft={{color: isAnonimityEnabled ? colors.white : '#648ABF'}}
-        />
-      </TouchableOpacity>
-    </View>
-  );
-};
+const {width} = Dimensions.get('screen');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `sendAnonymousDMOtherProfile`, `sendSignedDMOtherProfile` and `trimString` functions is removed. Ensure that the functionality provided by these functions is either no longer required or is being handled elsewhere in your application.
```diff
- import {sendAnonymousDMOtherProfile, sendSignedDMOtherProfile} from '../../service/chat';
- import {trimString} from '../../utils/string/TrimString';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
ok